telemetry/common.go

```diff
@@ -0,0 +1,51 @@
+package telemetry
+
+import (
+	cosmostelemetry "github.com/cosmos/cosmos-sdk/telemetry"
+	"github.com/hashicorp/go-metrics"
+)
+
+// MetricNameKeys constructs the full metric name by prefixing with a defined
```

Comment:
This isn't very clear.

1. Can you add an example in the comments?
2. Can you #PUC why it returns a slice and not a single string?